library/src/main/java/com/bumptech/glide/RequestManager.java

```diff
@@ -118,6 +118,7 @@ public RequestManager(
     glide.registerRequestManager(this);
   }
 
+  @NonNull
```

Comment:
This is a void method, it shouldn't need NonNull?

Reply:
Oh lol, Intellij didn't even complain. 😕